<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonthWindow</class>
 <widget class="QMainWindow" name="MonthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Análise do mês</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_yields">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>375</y>
      <width>941</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_month">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mês</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_transactions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>941</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_transactions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Transações</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_yields">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rendimentos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fixedIncome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rendas fixas</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_fixedIncome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>941</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
